SpatialGDK/Source/SpatialGDKServices/Private/SpatialCommandUtils.cpp

```diff
@@ -449,6 +380,38 @@ void SpatialCommandUtils::TryKillProcessWithName(const FString& ProcessName)
 	}
 }
 
+void SpatialCommandUtils::TryGracefullyKill(const FString& ProcName, const FProcHandle& ProcHandle)
+{
+#if !PLATFORM_WINDOWS
+	// On UNIX this sends a SIGTERM signal
+	FGenericPlatformProcess::TerminateProc(ProcHandle);
+#else
+	// TerminateProc is too forceful on Windows
+	TryGracefullyKillWindows(ProcName);
+#endif
+}
+
+void SpatialCommandUtils::TryGracefullyKillWindows(const FString& ProcName)
```

Comment:
Might be preferable to have the whole function in the #if PLATFORM_WINDWOS

You will need to add the check in the header as well.

Reply:
added!